docs/tasks/administer_cluster_quotas.md

```diff
@@ -210,6 +210,168 @@ To create the ClusterQueue, run the following command:
 kubectl apply -f cluster-total.yaml
 ```
 
+## Multiple ClusterQueues and borrowing cohorts
+
+Two or more ClusterQueues can borrow unused quota from other ClusterQueues in
+the same cohort.
+
+To establish a cohort, set a common name in the `.spec.cohort` field in all the
+ClusterQueues you want to include in the cohort.
+
+For two ClusterQueues, the manifests look like the following:
+
+```yaml
+# team-a-cq.yaml
+apiVersion: kueue.x-k8s.io/v1alpha1
+kind: ClusterQueue
+metadata:
+  name: team-a-cq
+spec:
+  namespaceSelector: {}
+  cohort: team-ab
+  requestableResources:
+  - name: "cpu"
+    flavors:
+    - resourceFlavor: default
+      quota:
+        guaranteed: 9
+        ceiling: 15
+  - name: "memory"
+    flavors:
+    - resourceFlavor: default
+      quota:
+        guaranteed: 36Gi
+        ceiling: 60Gi
+```
+
+```yaml
+# team-b-cq.yaml
+apiVersion: kueue.x-k8s.io/v1alpha1
+kind: ClusterQueue
+metadata:
+  name: team-b-cq
+spec:
+  namespaceSelector: {}
+  cohort: team-ab
+  requestableResources:
+  - name: "cpu"
+    flavors:
+    - resourceFlavor: default
+      quota:
+        guaranteed: 12
+  - name: "memory"
+    flavors:
+    - resourceFlavor: default
+      quota:
+        guaranteed: 48Gi
+```
+
+Note that the ClusterQueue `team-a-cq` also defines [ceilings](/docs/concepts/cluster_queue.md#ceilings).
+This restricts the ability of the ClusterQueue to borrow the unused quota from
+the cohort up to the configured ceiling, even if the quota is completely unused.
+
+You can create the ClusterQueues with the following command:
+
+```shell
+kubectl apply -f team-a-cq.yaml -f team-b-cq.yaml
+```
+
+## Multiple ClusterQueue with dedicated and fallback flavors
+
+A ClusterQueue can borrow resources from the cohort even if it has zero
+guaranteed quota for a flavor. This allows you to give dedicated quota for
```

Comment:
> A ClusterQueue can borrow resources from the cohort even if it has zero

In this sentence, what do you mean by "it"? I would suggest you repeat the noun to avoid confusions:
A ClusterQueue can borrow resources from the cohort even if the ClusterQueue  has zero guaranteed quota for a flavor (...)

Reply:
I was referring to the ClusterQueue indeed. Changed.